Pidaq/Examples/receiverInfo.json

```diff
@@ -0,0 +1,14 @@
+{
+  "canInfo": {
+    "serialPort": "COM6",
+    "busType": "slcan",
+    "bitrate": 500000,
+    "busFilters": [
```

Comment:
The python-can library uses hex values for can ID and can mask, but since python represents hex as a string, you pass the decimal representation of the hex value, and it does the conversion